pkg/question/question.go

```diff
@@ -60,13 +60,7 @@ func AskQuestion(input *AskQuestionInput) string {
 		fmt.Print(*input.OptionsString)
 	}
 
-	if input.DefaultOptionRepr != nil {
-		fmt.Printf("%s [%s]:  ", input.QuestionString, *input.DefaultOptionRepr)
-	} else if input.DefaultOption != nil {
-		fmt.Printf("%s [%s]:  ", input.QuestionString, *input.DefaultOption)
-	} else {
-		fmt.Printf(input.QuestionString + ": ")
-	}
+	GetQuestion(input)
```

Comment:
Instead of invoking this twice here and on line 118, I think it's equivalent and clearer to call it once after starting the for loop:

```
// Keep asking for user input until one valid input in entered
for {
  GetQuestion(input)
  ...
}
```

Or is this necessary to cover some edge case I'm missing?

Reply:
I agree with that. I think we can invoke it once at the start instead of invoking it twice, I missed it earlier. Will test and modify it. Thanks for the feedback :)